pfcpiface/up4.go

```diff
@@ -376,31 +357,17 @@ func (up4 *UP4) setUpfInfo(u *upf, conf *Conf) {
 	}
 }
 
-func (up4 *UP4) clearAllTables() error {
-	tables := []string{TableUplinkSessions, TableDownlinkSessions, TableUplinkTerminations, TableDownlinkTerminations, TableTunnelPeers, TableApplications}
-	tableIDs := make([]uint32, len(tables))
-
-	for _, table := range tables {
-		tableID, err := up4.p4RtTranslator.getTableIDByName(table)
-		if err != nil {
-			return err
-		}
-
-		tableIDs = append(tableIDs, tableID)
-	}
-
-	err := up4.p4client.ClearTables(tableIDs)
-	if err != nil {
-		return err
-	}
-
-	interfacesTableID, err := up4.p4RtTranslator.getTableIDByName(TableInterfaces)
-	if err != nil {
-		return err
+func (up4 *UP4) clearTables() error {
+	tableIDs := []uint32{
+		p4constants.TablePreQosPipeSessionsUplink,
+		p4constants.TablePreQosPipeSessionsDownlink,
+		p4constants.TablePreQosPipeTerminationsUplink,
+		p4constants.TablePreQosPipeTerminationsDownlink,
+		p4constants.TablePreQosPipeTunnelPeers,
+		p4constants.TablePreQosPipeInterfaces,
```

Comment:
`applications` table should also be cleared

Reply:
Done